frontend/common/src/messages/apiMessages.ts

```diff
@@ -150,6 +150,12 @@ export const messages: { [key: string]: MessageDescriptor } = defineMessages({
     description:
       "Error message that the pool advertisement must have publishing group filled.",
   },
+  APPLICATION_EXISTS: {
+    defaultMessage: "You have already applied to this pool",
+    description:
+      "Message displayed when a user attempts to apply to pool more than once",
+    id: "0OPWbJ",
+  },
```

Comment:
need one for NOT_PUBLISHED

Reply:
Good eye, added in [add localized message for unpublished pool](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4469/commits/3137b3feb8c7f13b3df316faea3b081376a88023)